pkg/clients/gitlab/git.go

```diff
@@ -120,3 +120,21 @@ func (gc *GitlabClient) CloseMergeRequest(projectID string, mergeRequestIID int)
 
 	return nil
 }
+
+// DeleteWebhooks delete webhooks in Gitlab repo by given project ID
+func (gc *GitlabClient) DeleteWebhooks(projectID string) error {
```

Comment:
How does this work with parallel tests runs? won't it interfere with other running tests?

Reply:
I'm deleting the webhook that the URL contains the console URL created the session [here](https://github.com/kasemAlem/e2e-tests/blob/0b7992a6bda43dbd8c10310705c1b10f42df5164/pkg/clients/gitlab/git.go#L126-L152) , wdyt?

Reply:
If we have another gitlab focussed test suite running in parallel, IIUC it might get affected by this webhook deletion since webhooks are set up per cluster.

We've had a similar problem in the past where 1 out of the 2 parallel suites deleted a cluster-scoped resource and that failed the other test suite. We ended up fixing that situation by moving the conflicting changes within the same test suite, maybe we would need to do the same thing here in the future.

Reply:
good point, 
I looked to see if I can a better identifier that the ClusterAppDomain , 
more like connect the MRIID with Webhook , but there is none.